content/blog/super-charging-our-github-action-with-the-pulumi-automation-api/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "GitHub Action now with added Pulumi Automation API"
+h1: "Super Charging our GitHub Action with the Pulumi Automation API"
+authors: 
+  - paul-stack
+#  - simen-olsen
```

Comment:
Will we have him listed before going live? Odd to have a "guest post" written by Paul Stack :-)

Reply:
yes we will :) It's a dual post - I'm waiting for some details from him